builder/src/bin/permissionless-builder.rs

```diff
@@ -25,12 +24,11 @@ pub struct NonPermissionedBuilderOptions {
     /// URL of the HotShot DA web server.
```

Comment:
This comment should also be updated. @move47

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1324/commits/8ed6b1806a1cccc8981efce30fbb888e3bc14882